network/manager.go

```diff
@@ -0,0 +1,569 @@
+package network
+
+import (
+	"encoding/binary"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/nknorg/nconnect/admin"
+	"github.com/nknorg/nconnect/config"
+	"github.com/nknorg/nkn-sdk-go"
+)
+
+const (
+	defaultDomain  = "nconnect.nkn"
+	defaultIpStart = "192.168.0.1"
+	defaultIpEnd   = "192.168.0.254"
+	defaultNetmask = "255.255.255.0"
+	defaultGateway = "192.168.0.1"
+	defaultDNS     = "192.168.0.1"
+
+	networkDataFile = "network.json"
+
+	AcceptAll = "all"

```

Comment:
I would suggest we use `AllMembers = "allMembers"`. There might be case when we have public nodes that accept connections from everyone (even not a member).

Reply:
Yes, modified.